### PR TITLE
refactor(VTabs): remove default colors workaround

### PR DESCRIPTION
**Pull request by fpdrozd**
## Description
This PR removes styles that were overwriting VTabs color prop.

## Motivation and Context
#10633

## How Has This Been Tested?
visually

## Markup:

<details>

```vue
<v-tabs color="yellow" background-color="primary">
  <v-tab>Tab 1</v-tab>
  <v-tab>Tab 2</v-tab>
</v-tabs>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VTabs/VTabs.sass [hunk 1]
```diff
@@ -16,18 +16,6 @@
 +theme(v-tabs-items) using ($material)
   background-color: map-get($material, 'cards')
 
-.v-tabs-bar
```
Comment (johnleider) [inline packages/vuetify/src/components/VTabs/VTabs.sass:19]: This is a breaking change.

File: packages/vuetify/src/components/VTabs/VTabs.ts [hunk 2]
```diff
@@ -183,6 +183,7 @@ export default baseMixins.extend<options>().extend({
     },
     genBar (items: VNode[], slider: VNode | null) {
       const data = {
+        staticClass: this.backgroundColor,
```
Comment (johnleider) [inline packages/vuetify/src/components/VTabs/VTabs.ts:186]: What is this change for?

**Comment by fpdrozd** [comment]
@johnleider This PR is supposed to revert [your commit](https://github.com/vuetifyjs/vuetify/commit/cb1c61482dd3389c67552e190b28cd684f5eefd5) that was causing this problem. You said it was some sort of a workaround and can be removed now.

**Comment by johnleider** [comment]
Correct, but that commit is from July of last year. If it's not resolvable without breaking changes, we'll have to move it to v2.3. Let me know.

**Comment by fpdrozd** [comment]
Maybe we could check that in `computedColor`. Something like this:
```
computedColor (): string {
  if (this.color) return this.color
  else if (
    this.backgroundColor in preset.theme.themes.light ||
    (this.isDark && !this.appIsDark)
  ) return 'white'
  else return 'primary'
},
```

**Comment by johnleider** [comment]
This Pull Request is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Pull request closed without merging**